packages/modals/src/elements/Modal.tsx

```diff
@@ -71,6 +71,14 @@ export interface IModalProps {
    * Enable large modal styling
    */
   isLarge?: boolean;
+  /**
+   * Determines whether to put keyboard focus on the modal on mount
```

Comment:
> to ~put~ apply keyboard

Does this make any sense?

Reply:
Yeah that wording makes sense 👍, thanks!